config/sources/families/include/rockchip64_common.inc

```diff
@@ -56,6 +56,8 @@ esac
 # - tpl-spl-blob: uses mainline u-boot TPL and SPL with proprietary rockchip ATF blob
 # - tpl-blob-atf-mainline: proprietary rockchip ddrbin + mainline u-boot SPL + mainline ATF
 # - blobless: mainline u-boot TPL + mainline u-boot SPL + mainline ATF
+# - binman:  u-boot builds full boot image from information in device tree.  See:
+#               https://github.com/ARM-software/u-boot/blob/master/tools/binman/README
```

Comment:
This link to wrong repo, you could link to U-Boot docs:
https://docs.u-boot.org/en/latest/develop/package/binman.html

Reply:
Corrected the URL.  Pushed update.